src/server/scripts/EasternKingdoms/ZulAman/boss_akilzon.cpp

```diff
@@ -114,8 +114,9 @@ struct boss_akilzon : public BossAI
                 EnterEvadeMode();
                 return;
             }
-            target->CastSpell(target, SPELL_ELECTRICAL_STORM_AREA, true); // cloud visual
+
             DoCast(target, SPELL_ELECTRICAL_STORM); // storm cyclon + visual
+            target->CastSpell(target, SPELL_ELECTRICAL_STORM_AREA, true); // cloud visual
```

Comment:
I don't have a storm sniff :cry:  but found this:

https://github.com/cmangos/mangos-wotlk/commit/c4f4839caa9545a573570d66024e9b3ad73488de

Can this be done as a 500ms offset as part of
```
 ID - 43648 Electrical Storm
Effect 2: Id 77 (SPELL_EFFECT_SCRIPT_EFFECT)
```
instead of casting 44007 here,  or will the visual be delayed this way?

```
        OnEffectHitTarget += SpellEffectFn(spell_electrial_storm::HandleScript, EFFECT_2, SPELL_EFFECT_SCRIPT_EFFECT);
        ...
                caster->m_Events.AddEventAtOffset([caster]()
                {
                        caster->CastSpell(caster, 44007, true);
                }, 500ms);
```

Reply:
Hmm, thanks